packages/babel-runtime/helpers/builtin/toArray.js

```diff
@@ -1,3 +1,5 @@
+"use strict";
```

Comment:
seems unrelated but ok

Reply:
It was auto inserted when I ran `make fix`. I missed it while committing, my bad.

Reply:
They're in my PRs now, too.

Reply:
oh it must be after @loganfsmyth's helpers PR - maybe modules/import related?

Reply:
Oh woops, that's from https://github.com/babel/babel/commit/2801bfe35c55e430f5348b9cab3bf860aef8b4e7, looks like we were using it in https://github.com/babel/babel/blob/master/packages/babel-runtime/scripts/build-dist.js#L63

Let me push a fix and remove these changes.

Reply:
Fixed https://github.com/babel/babel/commit/f35cf8185b542d39293dbb35aa6e07dd18fcd668, so you can remove these changes from the PR now.

Reply:
Updated the PR.